### ISSUE TITLE
ScheduledExecutorServiceTest.stats_longRunningTask_durable

### ISSUE THREAD

**Issue by hasancelik** [hazelcast/hazelcast#9492]
```
java.lang.AssertionError: expected:<6> but was:<7>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.hazelcast.scheduledexecutor.ScheduledExecutorServiceTest.stats_longRunningTask_durable(ScheduledExecutorServiceTest.java:234)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-ZuluJDK6/com.hazelcast$hazelcast/420/testReport/junit/com.hazelcast.scheduledexecutor/ScheduledExecutorServiceTest/stats_longRunningTask_durable/

**Comment by hasancelik** [comment]
```
java.lang.AssertionError: expected:<6> but was:<5>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.hazelcast.scheduledexecutor.ScheduledExecutorServiceTest.stats_longRunningTask_durable(ScheduledExecutorServiceTest.java:234)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-IbmJDK1.8/com.hazelcast$hazelcast/445/testReport/junit/com.hazelcast.scheduledexecutor/ScheduledExecutorServiceTest/stats_longRunningTask_durable/

**Comment by jerrinot** [comment]
https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x/com.hazelcast$hazelcast-client/6247/testReport/junit/com.hazelcast.client.scheduledexecutor/ClientScheduledExecutorServiceTest/stats_longRunningTask_durable/

```
java.lang.AssertionError: expected:<6> but was:<5>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.hazelcast.scheduledexecutor.ScheduledExecutorServiceTest.stats_longRunningTask_durable(ScheduledExecutorServiceTest.java:234)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:88)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:82)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.lang.Thread.run(Thread.java:662)
```

**Comment by tkountis** [comment]
Suggested fix #9580

**Comment by tkountis** [comment]
#9580 merged.
